[export] Make unflattened module compileable

angelayi opened the pull request:
Test Plan: Fixes https://fb.workplace.com/groups/1028545332188949/permalink/1091988579177957/

Differential Revision: D66302806

avikchaudhuri reviewed (approved):
Is it just the pytree call that couldn't be compiled?

avikchaudhuri commented:
also lol at the test failures, can't fix spelling in peace

angelayi committed:
[export] Make unflattened module compileable (#141249)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/141249

Test Plan: Fixes https://fb.workplace.com/groups/1028545332188949/permalink/1091988579177957/

Reviewed By: avikchaudhuri

Differential Revision: D66302806
